src/Classifier.php

```diff
@@ -6,145 +6,86 @@
 
 use ReflectionAttribute;
 use ReflectionClass;
-use Symfony\Component\Finder\Finder;
 
-final class Classifier
+final class Classifier extends AbstractClassifier
```

Comment:
I'm thinking about rename it to `PhpNativeClassifier` or w/o Php prefix. WDYT @yiisoft/yii3 ?

Reply:
Need rename more specific than just `Classifer`. You can creare issue and make it in separate PR.